app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.js");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
please remove this and re-add when posting/patching. 😸

Reply:
okay!